doc/api/https.md

```diff
@@ -288,6 +289,10 @@ The following additional `options` from [`tls.connect()`][] are also accepted:
 string, it is automatically parsed with [`new URL()`][]. If it is a [`URL`][]
 object, it will be automatically converted to an ordinary `options` object.
 
+`https.request()` returns an instance of the [`http.ClientRequest`][]
+class. The `ClientRequest` instance is a writable stream. If one needs to
```

Comment:
Pronouns are OK in our docs, so if you want to use `you` instead of `one`, that would be fine:

```suggestion
class. The `ClientRequest` instance is a writable stream. If you need to
```